crun.1

```diff
@@ -707,9 +707,74 @@ These flags are supported:
 
 .SH idmap mount options
 .PP
-If the \fB\fCidmap\fR option is specified then the mount is ID mapped using the container
-target user namespace.  This is an experimental feature and can change at any time
-without notice.
+If the \fB\fCidmap\fR option is specified then the mount is ID mapped using
+the container target user namespace.  This is an experimental feature
+and can change at any time without notice.
+
+.PP
+The \fB\fCidmap\fR option supports a custom mapping that can be different
+than the user namespace used by the container.
+
+.PP
+The mapping can be specified after the \fB\fCidmap\fR option like:
+\fB\fCidmap=uids=0-1-10;gids=0-100-10\fR\&.
+
+.PP
+For each triplet, the first value is the ID inside the container, the
+second is the ID on the host and the third one refers to the range
```

Comment:
For a normal user namespace, I would agree with that description but for a user namespace that is used for a MOUNT_ATTR_IDMAP I would phrase it differently since it is mapping from the perspective of the filesystem. Specifically "ID inside the container" seems wrong to me since it is actually the id on the backing filesystem that gets mapped.

This [ug]id_map would map on disk gid/uid 100000 to gid/uid  0.
100000 0  1

Reply:
Maybe something like this?
For each triplet, the first value is the start of the backing filesystem ids that are mapped to the second value on the host. The length of this mapping is given in the third value.

Reply:
sure, that looks better.

Amended and pushed a new version